frontend/plugins/accounting_ui/src/modules/transactions/transaction-form/components/forms/InvIncomeForm/index.tsx

```
@@ -38,10 +45,49 @@ export const InvIncomeForm = ({
+
+  useEffect(() => {
+    const proportionAmount = sumAmountExpenses / sumAmountDetails;
+    const proportionCount = sumCountExpenses / sumCountDetails;
+
+    details.forEach((detail, detIndex) => {
+      const amount = fixNum((detail.count ?? 0) * (detail.unitPrice ?? 0));
+      const newAmount = amount + fixNum(proportionCount * (detail.count ?? 0)) + fixNum(proportionAmount * amount);
+      form.setValue(`trDocs.${journalIndex}.details.${detIndex}.amount`, newAmount);
+    });
+  },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)